algorithms/dijkstra/README.md

```diff
@@ -0,0 +1,50 @@
+# Dijkstra Shortest Path
+
+Dijkstra's algorithm is an algorithm for finding the shortest paths between nodes in a graph.
+
+### Input Format
```

Comment:
Add a more descriptive input format.

Reply:
@aryadas98 I have added more information to the input format.